gonum/README.md

```diff
@@ -4,7 +4,9 @@
 ## What is this image?
 
 `clearlinux/gonum` is a Docker image with `gonum` running on top of the
-[official clearlinux base image](https://hub.docker.com/_/clearlinux). 
+[official clearlinux base image](https://hub.docker.com/_/clearlinux). User
+can deploy gonum apps on top of `clearlinux/gonum` container, and get better
+performance via newlib interface which call clearlinux avx512 optimized CBLAS. 
```

Comment:
Do we need state that the performance gain can only got on AVX512 available platform?

Reply:
Performance data is always sensative to share. And the gain should keep consistent with clearlinux official perf data.